.codeclimate.yml

```diff
@@ -1,3 +1,5 @@
 exclude_patterns:
-- "tests/"
-- "lib/"
\ No newline at end of file
+- "src/"
+- "spherov2.js-lib/tests/"
```

Comment:
I guess you have to rename this too

Reply:
But, wait, are we excluding everything?

Reply:
Oh sorry, you are right ... I just forgot to change this. It's fixed now.